### DISCUSSION TITLE
blog/210924-bithacks-power/

### DISCUSSION THREAD

**Discussion by giscus** [evandde/blog_comment#41] [Announcements]
# blog/210924-bithacks-power/

자연수 승수 연산에 한해, 비트연산 트릭을 활용하여 C++에서 기본으로 제공하는 pow() 함수보다 빠르게 계산하는 방법을 알아봅니다.

https://evanote.mew.kr/blog/210924-bithacks-power/

**Comment by lsm3861** [comment]
박사님 글 잘 읽었습니다.
거의 2배정도 빨라지네요...
